src/core/js/drawer.js

```diff
@@ -25,7 +25,12 @@ define(function(require) {
 	Adapt.once('app:dataReady', function() {
 		init();
 	})
-
+	Adapt.on("drawer:opened",function(){
+		var itemsLength = $(".drawer-holder")[0].childElementCount;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/ParentNode/childElementCount

^ childElementCount not supported in ie8. can you find a different way to do this?